The job **`x86_64-gnu-llvm-14`** failed! Check out the build log: [(web)](https://github.com/rust-lang/rust/actions/runs/5328493010/jobs/9653155504) [(plain)](https://github.com/rust-lang/rust/commit/21d9fd77f4e163dbf1124241a77e4ccbd302ac20/checks/14421469559/logs)

<details><summary><i>Click to see the possible cause of the failure (guessed by this bot)</i></summary>

```plain
   Compiling mbe v0.0.0 (/checkout/src/tools/rust-analyzer/crates/mbe)
error[E0432]: unresolved import `proc_macro::LineColumn`
  --> crates/proc-macro-srv/src/server.rs:13:5
   |
13 |     LineColumn,
   |     ^^^^^^^^^^ no `LineColumn` in the root
For more information about this error, try `rustc --explain E0432`.
error: could not compile `proc-macro-srv` (lib test) due to previous error
Build completed unsuccessfully in 0:29:11
```

</details>

--- a/src/tools/rust-analyzer/crates/proc-macro-srv/src/server.rs
+++ b/src/tools/rust-analyzer/crates/proc-macro-srv/src/server.rs
@@ -8,10 +8,7 @@
 //!
 //! FIXME: No span and source file information is implemented yet
 
-use proc_macro::{
-    bridge::{self, server},
-    LineColumn,
-};
+use proc_macro::bridge::{self, server};
 
 mod token_stream;
 pub use token_stream::TokenStream;

